### PR TITLE
Add state footer for comments and replies pages

### LINKED ISSUE #23: [FEATURE REQUEST] State Footer for Comments and Replies Pages
### Is your feature request related to a problem? Please describe.
As a user, I would like to be able to test multiple endpoints of the demo in a very simple and easy to understand navigation for switching between different tests.

### Describe the solution you'd like
A react smart component should use the required dumb components to create a functional footer that holds `View Current State` and `Reset State` buttons.

### Describe alternatives you've considered
None.

### Additional context
See the Home Page frame in the [figma prototype](https://www.figma.com/file/syIGWYjqpOqikiXM55fuEO/Comment-API-Demo?node-id=17%3A49) for the demo design.

### PR DESCRIPTION
**Pull request by piouson**
Resolve #23

**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Resolve #23 

**description of Task to be completed?**

Add `StateFooter` that holds buttons for `View Current State` and `Reset State`

**How should this be manually tested?**

**Any background context you want to provide?**

**Questions:**

### CODE REVIEW

File: src/components/StateFooter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Button, Grid, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+    maxWidth: 1000
```
Comment (simply-alliv) [inline src/components/StateFooter/index.tsx]: This seems arbitrary. Material UI uses a grid system that increments by 8 to maintain consistency. You should always use numbers divisible by 8.

File: src/components/StateFooter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  },
+  button: {
+    textTransform: "none"
+  }
```
Comment (simply-alliv) [inline src/components/StateFooter/index.tsx]: Why did you need this for the button?

File: src/components/StateFooter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          <Button className={classes.button} onClick={handleViewState}>
+            View Current State
+          </Button>
+        </Grid>
+        <Grid item>
+          <Button className={classes.button} onClick={handleResetState}>
+            Reset State
+          </Button>
+        </Grid>
+    </Grid>
```
Comment (simply-alliv) [inline src/components/StateFooter/index.tsx]: A grid is not necessary for one row. Rather use the Box component with a display of flex and justifyContent  of space-between. It's much simpler. Always use Box first before using other layout components.

**Review by simply-alliv — CHANGES_REQUESTED** [review]

File: src/components/StateFooter/index.tsx (hunk 2 above)
Comment (piouson) [reply src/components/StateFooter/index.tsx]: The button text defaults to all caps, so removed with `textTransform`

Comment (simply-alliv) [reply src/components/StateFooter/index.tsx]: I see. That's actually a good idea. Better yet, why don't you place this property in the global `src/theme/index.ts theme file. That way all buttons will get this. What do you think? Place the exact same object inside the typography object 
```suggestion
  .. // other theme properties
  typography: {
    ... 
    button: {
      textTransform: "none"
    }
  }
```

File: src/components/StateFooter/index.tsx (hunk 3 above)
Comment (simply-alliv) [reply src/components/StateFooter/index.tsx]: If you are interested: 

- [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
- https://material-ui.com/system/flexbox/

File: src/components/StateFooter/index.tsx (hunk 1 above)
Comment (piouson) [reply src/components/StateFooter/index.tsx]: I suppose maxWidth can be handled by App or parent other parent container and I remove from here..?

Comment (simply-alliv) [reply src/components/StateFooter/index.tsx]: Yeah, this might end up causing unwanted inconsistencies with alignment and other positions.

File: src/components/StateFooter/index.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  const handleViewState = (event: any) => {};
+
+  const handleResetState = (event: any) => {};
+  
+  return (
+    <Box className={classes.root}>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="flex-end"
```
Comment (simply-alliv) [inline src/components/StateFooter/index.tsx]: Whether you add or remove this. It probably doesn't make a difference. So if it doesn't, you can remove it. There's no need to increase the files byte size.
